pilot/pkg/xds/internalgen.go

```diff
@@ -52,12 +54,27 @@ const (
 type InternalGen struct {
 	Server *DiscoveryServer
 
-	Store model.ConfigStore
+	// Store only tracks things in k8s since we don't handle writing to MCP based stores or reconciling across different store types.
```

Comment:
Comment is inaccurate - uses k8s when running in real pilot, uses in-memory for tests.